src/keylessBackup/slice.ts

```diff
@@ -1,11 +1,13 @@
 import { PayloadAction, createSlice } from '@reduxjs/toolkit'
+import { KeylessBackupStatus, KeylessBackupFlow } from 'src/keylessBackup/types'
 
 export interface State {
   google: {
     status: 'idle' | 'loading' | 'success' | 'error'
     idToken: string | null
   }
   valoraKeyshare: string | null
+  backupStatus: KeylessBackupStatus | null
```

Comment:
is this a state that we'll want to persist and identify cab status on the rest of the app? Or is it just a temporary state to identify the status in backup status screen? If it's the former, I guess we'd have to persist and currently the entire keylessBackup state is blacklisted from redux-persist

Reply:
we want to know if a user has backed up yet for the side menu and in settings

Reply:
Satish and I discussed this offline. We decided that this state would be ephemeral (session-only), and we'll add a _new_ piece of state to an existing, _persisted_ object in Redux to denote whether or not the user has completed CAB. This makes life a lot easier, since we won't have to be selective about which pieces of state in the `keylessBackup` object get persisted, which is something we have not done before.